Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM eclipse-temurin:21
+
+ARG JAR_FILE=build/libs/*.jar
+
+WORKDIR /app
+
+ENV JAVA_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```

Comment:
why we need this ? Can it be simpler 
FROM openjdk:8-jdk-alpine
ARG JAR_FILE=target/*.jar
COPY ${JAR_FILE} app.jar
ENTRYPOINT ["java","-jar","/app.jar"]

https://spring.io/guides/gs/spring-boot-docker

Reply:
you can see the same here: 
![image](https://github.com/user-attachments/assets/6f576a45-9d05-4bac-9057-58acf98c840b)

I just want to make easier and have always available while we are actively making changes. But we can remove later when we have an stable version.